integration_test/integration_test.go

```diff
@@ -360,6 +360,93 @@ projects:
 	assert.Error(t, err, "modified file did not trigger verify fail")
 	stdout := outputBuf.String()
 	assert.True(t, strings.Contains(stdout, structsFile+": checksum changed"), "Unexpected standard out: %s", stdout)
+
+	t.Run("verify does not detect stale files that would not be deleted", func(t *testing.T) {
```

Comment:
Make these new test functions or have an outside test function that calls inner `t.Run` functions more consistently -- conflating these different test cases in this current manner doesn't really make sense